kqueen/blueprints/api/views.py

```diff
@@ -220,6 +220,27 @@ def cluster_resize(pk):
     return jsonify(output)
 
 
+@api.route('/clusters/<uuid:pk>/set_network_policy', methods=['PATCH'])
+@jwt_required()
+def cluster_set_network_policy(pk):
+    obj = get_object(Cluster, pk, current_identity)
+
+    data = request.json
+    if not isinstance(data, dict) or (isinstance(data, dict) and 'node_count' not in data):
```

Comment:
> or (isinstance(data, dict) and 'node_count' not in data)
> isinstance(data, dict)
This is redundant, because you will not reach this condition if otherwise

Reply:
why not? pls clarify info

Reply:
```
not x or (x and y) = not x or y

x | y | not x | x and y | not x or (x and y) | not x or y
0 | 0 |   1   |    0    |         1          |      1    
0 | 1 |   1   |    0    |         1          |      1    
1 | 0 |   0   |    0    |         0          |      0    
1 | 1 |   0   |    1    |         1          |      1    
```
Hope this is visually enough

Reply:
Or `not x or (x and y) = not x or x and not x or y = True and not x or y = not x or y`